### ISSUE TITLE
Bottom Y braces (and the holes in the panels) standard in build

### ISSUE THREAD

**Issue by dandancheeseandham** [railcore/parts#7]
They seem to be a popular add-on.

By adding holes in the bottom panel in F360 for the brace, it sets a standard distance, laser cut or milled precisely so the extrusion is definitely straight.

That way also, additional mods such as https://www.thingiverse.com/thing:3629939 can also be standardised somewhat.

**Comment by lostapathy** [comment]
I agree - even if @Project-R3D doesn't want to include the additional extrusions in the kit, it'd be great to get the holes cut so they are in a standardized location.

I am biased because I designed the part linked above, but I think the spacing it relies on is ideal and more or less the "standard" people who have done it by hand have shot for as well.

**Comment by Project-R3D** [comment]
We are working with the RailCore Labs Team on our mid year update which will include these extra extrusions in the kits.

**Comment by lostapathy** [comment]
Awesome!  Can we get the layout documented sooner rather than later so that people retrofitting can be compatible?  Preferably based on the above-linked part if it's not yet set in stone.
